CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.6.2
+- Setting up sticky video controls for Scene Viewer
+- Upgrade aws-sdk and iot-app-kit 
```

Comment:
Need to specify the exact changes here as these are exposed to customers.

- Setting up sticky video controls for Video Player
- Error handling for Matterport scenes
- Auto collapse sidebar for Scene Viewer
- Fix usage of repeat template variables for Scene Viewer and Video Player panels
- Other minor bug fixes in Scene Viewer and Video Player panels

Reply:
Fixed! Thank you :)